### DISCUSSION TITLE
Access restrictions between clients [swanctl.conf]

### DISCUSSION THREAD

**Discussion by acrolink** [strongswan/strongswan#629] [Q&A] [answered]
I assign to clients specific IP addresses based on EAP identities (using pools). Is it possible for instance to:

Make '10.50.10.10' reach '10.50.10.20' but prevent the latter from reaching the former?

i.e. for instance allow '10.50.10.10' to ping '10.50.10.20' but not the other way around?

**Comment by tobiasbrunner** [comment, marked as answer]
Just configure firewall rules in the `FORWARD` chain accordingly (you could configure the default policy to `DROP` and then only allow specific traffic - if only in one direction, use `-m state` to allow `NEW` states only in one direction and `ESTABLISHED,RELATED` in both).

**Reply by acrolink** [reply]
@tobiasbrunner , you're a lifesaver. Thank you very much. Thank you all for this great project.
